src/Sylius/Bundle/ApiBundle/Controller/TerminateCatalogPromotionAction.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Controller;
+
+use Sylius\Bundle\CoreBundle\CatalogPromotion\Command\TerminateCatalogPromotion;
+use Sylius\Component\Core\Model\CatalogPromotionInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+/** @experimental */
+final class TerminateCatalogPromotionAction
```

Comment:
We use "removal" in scenarios, but "terminate" here. Cannot it be `RemoveCatalogPromotionAction`?